packages/terra-status-view/tests/jest/__snapshots__/StatusView.test.jsx.snap

```diff
@@ -0,0 +1,331 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should render an image with a custom status-view 1`] = `
+<StatusView
+  isAlignedTop={false}
+  isGlyphHidden={false}
+  variant="custom"
+>
+  <div
+    className="status-view"
+  >
+    <div
+      className="status-view-inner"
+      style={
+        Object {
+          "paddingBottom": "0px",
+          "paddingTop": "0px",
+        }
+      }
+    >
+      <div
+        className="glyph"
+      >
+        <svg
+          className="custom"
+        />
+      </div>
+      <div
+        className="message-content-group"
+      >
+        <div
+          className="title"
+        />
+      </div>
+    </div>
+  </div>
+</StatusView>
+`;
+
+exports[`should render an image with a no-data status view that will hide the glyph from being shown 1`] = `
+<StatusView
+  isAlignedTop={false}
+  isGlyphHidden={true}
+  variant="no-data"
+>
+  <div
+    className="status-view"
+  >
+    <div
+      className="status-view-inner"
+      style={
+        Object {
+          "paddingBottom": "0px",
+          "paddingTop": "0px",
+        }
+      }
+    >
+      <div
+        className="message-content-group"
+      >
+        <div
+          className="title"
+        >
+          No Results
+        </div>
+      </div>
+    </div>
+  </div>
+</StatusView>
+`;
+
+exports[`should render an image with a no-data status view with all message props given 1`] = `
+<StatusView
+  isAlignedTop={false}
+  isGlyphHidden={false}
+  message="Test Message"
+  title="Test Title"
+  variant="no-data"
+>
+  <div
+    className="status-view"
+  >
+    <div
+      className="status-view-inner"
+      style={
+        Object {
+          "paddingBottom": "0px",
```

Comment:
Would you expect these padding values to be 0 here?

Reply:
Or what is the case these wouldn't be 0? I am curious on getting a screenshot of an instance where the padding is calculated

Reply:
this is for when the status view has a default height that is larger than what its content is.  It will add padding after mounting to align the content in the middle with 40% on the top and 60% on the bottom of the remaining space.

Reply:
I think it would be good to add a test case to verify this is calculated & applied as expected